### PR TITLE
Add Feature: Responsive Mobile to Desktop

### PR DESCRIPTION
**Pull request by miguelesco**
Add:

- Responsive Features for the Headline and new Background Image.
- Changes to the HTML Structure of the Headline.
- Change the Arrow Button from the Works Section to the Headline Section.
- Media queries for the Headline, Works, Bio, and Contact Section.
- Moved the list of accounts from the Contact section to a footer.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -341,3 +369,124 @@ textarea {
   color: #42526e;
   line-height: 32px;
 }
+
+@media screen and (min-width: 750px) {
```
Comment (Davidosky007) [inline style.css]: * This min-width value should serve minimum width for the desktop version and better to make it in the range 992px-1100px. Yours is a bit smaller than standard desktop widths.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @gabrielyea @miguelesco, 

Good job so far you have done a good 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!
cr2 gif_almost-there


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 1 above)
Comment (henatan99) [reply style.css]: - This issue is well addressed! :clap:

File: style.css [hunk 2]
```diff
@@ -341,3 +369,124 @@ textarea {
...
+    margin: auto;
+    margin-top: 20vh;
+    background-image: url('./assets/ilustration-left-contact.svg'), url('./assets/ilustration-contact.svg'), url('./assets/ilustration-right-contact.svg');
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: left bottom, 25% 65%, right;
+  }
+
+  #contact h3 {
+    flex: 50%;
+    text-align: justify;
```
Comment (henatan99) [inline style.css]: - [x] OPTIONAL: I don't think this `text-align: justify` is a good styling for the text here. The original design looks like `text-align: left`. Please compare the attached screenshot images of yours and original design. 

image   
image

File: index.html [hunk 3]
```diff
@@ -205,6 +250,9 @@ <h3>
                 </div>
             </form>
 
+        </section>
+
+        <footer>
             <ul class="socialLinks">
```
Comment (henatan99) [inline index.html:256]: - [x] OPTIONAL: The social links element has a gray line on top of it. Kindly implement it in your spare time. 
image

**Review by henatan99 — APPROVED** [review]
Hi @miguelesco , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**